include/irr/builtin/glsl/bxdf/brdf/specular/beckmann.glsl

```diff
@@ -151,7 +165,7 @@ vec3 irr_glsl_beckmann_cos_remainder_and_pdf(out float pdf, in irr_glsl_LightSam
     }
     else
     {
-        pdf = 0.0;
+        pdf = irr_glsl_beckmann_pdf(interaction, _cache, a2);
```

Comment:
pdf should be 0.0, because either the viewer or light is behind the surface and this is a BRDF

Reply:
my bad, if it actually gets past an `is_Valid` check (the checks of microfacet orientation) for a BRDF...

then we need to compute the PDF.

BUT dont branch and compute the same PDF functions in both branches! UNIFY!

Reply:
TODO